README.md

```diff
@@ -1,2 +1,74 @@
+[Abbreviations](#Abbreviations)  
+[Capitalization](#Capitalization)  
+[Acronyms](#Acronyms)  
+[Commas in Lists](#Commas-in-Lists)  
+[Numbers](#Numbers)  
+[Contractions](#Contractions)  
+[Apostrophes](#Apostrophes)   
+## Bold  
+Use bold to highlight button names, fields names, menus, radio buttons, or checkboxes. For example:  
+- Click the **Create** button to start the process.  
+- Enter your IP address in the **IP Address** field.  
+
+## Italics
+Use italics to emphasize a word or group of words. For example:
+- At least two reviews are required before you check in your code.
+- You must not enter without your ID card.  
+> **Important:** Use this style sparingly to avoid diluting it's strenght.  
+  
+  ## Fixed font  
+  Use a fixed font to highlight:
+  - **Commands or pieces of code.** For example:
+Run the ```npm install text-cli``` command to install the utility.  
+- **File or folder names.** For example:
+Read the `README.md` file before you proceed  
+## Abbreviations
+Never use the following abbreviations in a formal document:  
+|   |    |
+| -----------  | ----------- |
+| e.g. | Instead use "for example". |
+| i.e.  | Instead use "in other words". |  
+| etc.  | Instead give the full list of optionis. |
+| fig.  | Instead spell it out as "figure". |
+| pg. | Instead spell it ouf as "page". |  
+  
+    
+## Capitalization  
+Use standard capitalization rules. Also, keep in mind the following:  
+- The first word after a colon (:) is always capitalized.  
+- Every heading, table caption, figure caption, column name, or a callout in a figure or table
+must be in the title case.  
+## Acronyms  
+Always define an acronym when you are using it for the first time.
+For example: This section specifies the terms used in a Product Requirement Document (PRD).
+The most often used terms in a PRD include…  
+## Commas in Lists  
+The rules regarding commas for in-sentence lists are as follows:  
+- Do not use a comma before “and” and “or” if your list contains only two items. For example: Any new organization needs to invest in good infrastructure and hiring great talent.  
+- Use a comma before “and” and “or” when your list comprises three or more items. For example: A “Pod” comprises a Project Manager, a Technical Writer, Engineers, and UX designers.  
+
+## Numbers
+Remember these rules about numbers:  
+- Spell out numbers that are less than 10 (0 - 9). For example: I have seven aunts.  
+- Use numerals for 10 and higher. For example: This hub has 32 ports.  
+- Use a comma to make it easier to read numbers higher than 999.
+For example: The network supports up to 10,000 connections.  
+
+## Contractions
+Contractions are considered informal writing. To prevent “stuffiness” use the contraction “don’t”. Avoid other contractions such as “it’s” or “doesn’t”.  
+
+## Apostrophes  
+
+Do not use an apostrophe for the plural of acronyms. For example:
+|  | |
+|---  | ---|
+|API's  | Instead use APIs |
+|LAN'S  | Instead use LANs
```

Comment:
Missing the closing pipe.

Reply:
Pipe added. 
Added a header to the table.